src/routes/+page.svelte

```diff
@@ -1,13 +1,169 @@
+
+    // Functions
+    // Runs as soon as the component is mounted
+    onMount(async () => {
+        planData = new Promise(async (resolve, reject) => {
+            try {
+                // Fetch backend for plans
+                const response = await tryFetchingPublic(urls.get.allPlans);
+                if (response.ok) {
+                    let plans = await response.json();
+                    resolve(
+                        plans.sort((planA, planB) => {
+                            return planA.monthlyPrice - planB.monthlyPrice;
+                        }),
+                    );
+                } else {
+                    throw new Error("Error while fetching data");
+                }
+            } catch (err) {
+                reject(err); // Rethrow so Svelte can handle it
+            }
+        });
+    });
 </script>
 
 <svelte:head>
     <title>Voycar - Startseite</title>
 </svelte:head>
 
-<div class="flex h-[70vh] flex-col items-center justify-center">
-    <div class="border-md rounded-md bg-surface-700 p-4">
-        <h1 class="h1 text-center">Willkommen bei Voycar</h1>
+<section class="flex h-full flex-col items-center justify-center">
+    <div class="grid max-w-screen-xl lg:grid-cols-12">
+        <div class="place-self-center lg:col-span-7">
+            <h1 class=" h1 mb-8 xl:text-6xl">
+                Minimalistisch unterwegs. Maximal grün.
+            </h1>
+            <p class=" h4 mb-4">Dein CarSharing für alle Fälle.</p>
+            <div>
+                {#if !$loggedIn}
+                    <div class="flex flex-row space-x-2">
+                        <button
+                            class="variant-ringed-surface btn"
+                            on:click={() => {
+                                goto("/register");
+                            }}>Registrieren</button
+                        >
+                        <button
+                            class="variant-ringed-primary btn"
+                            on:click={() => {
+                                goto("/login");
+                            }}>Anmelden</button
+                        >
+                    </div>
+                {/if}
+            </div>
+        </div>
+        <div
+            class="hidden items-center justify-center lg:col-span-5 lg:mt-0 lg:flex"
+        >
+            <img
+                src="/Range%20Rover.png"
+                alt="logo"
+                class="h-auto w-[800px] scale-100"
+                style="max-width: 100%; height: auto;"
+            />
+        </div>
     </div>
-</div>
+    {#if !$loggedIn}
+        <div class="sm:py-22 mt-20 py-10">
+            <div class="mx-auto max-w-7xl px-6 lg:px-8">
+                <div class="sm:text-center">
+                    <p
+                        class="mt-2 text-4xl font-bold tracking-tight sm:text-5xl"
+                    >
+                        Wähle das richtige Abo für dich
```

Comment:
![grafik](https://github.com/user-attachments/assets/ccb0c539-04e4-43ec-bb29-42392e2293ae)
Hier wählt man nicht. Passender wäre "Für alle das richtige Abo" oder so

Reply:
Ich finde die Formulierung jetzt nicht wirklich schlimm. 
Man hat aber doch die Auswahl zwischen 3 Abomodellen. "Wähle" bezieht sich nicht auf den imperativ hier, sondern darauf, dass man wählen kann, wie bspw. in Werbungen gesagt wird "Wähle zwischen drei Geschmackssorten [...]" da wählst du ja auch nicht in dem Moment, sondern du hast dann, wenn du das Produkt kaufst (oder wie in unserem Fall deine Reservierung abschließt) die Wahl dich für eines der drei Abomodelle zu entscheiden. 
Aber an sich ist mir das egal, wenn du/ihr eine passendere Formulierung habt, dann können wir auch die nehmen.